### PR TITLE
SGD8-629: Fixed bug with checkboxes in modal-actions

### PR DESCRIPTION
**Pull request by hstandaert**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [x] I have updated the styleguide CHANGELOG accordingly.

### CODE REVIEW

**Review by delrueba — COMMENTED** [review]
Might be useful to start collection the z-indexes in variables.
So we know where the modals, mobile menu, lightbox.... chat(?) are located

File: components/31-molecules/modal/_modal.scss [hunk 1]
```diff
@@ -5,6 +5,7 @@
   right: 0;
   bottom: 0;
   left: 0;
+  max-height: 100vh;
```
Comment (delrueba) [inline components/31-molecules/modal/_modal.scss:14]: Have you checked on mobile?
This gives issues with the URL toolbar.

**Review by delrueba — CHANGES_REQUESTED** [review]
see comments

File: components/31-molecules/modal/_modal.scss (hunk 1 above)
Comment (hstandaert) [reply components/31-molecules/modal/_modal.scss:14]: Well, I've tested it on multiple OS / browsers:

* Android (Chrome)
* iOS (Chrome / Safari)
* OSX (Chrome / Safari / Firefox)
* Windows (IE 11)

And weirdly enough, it does the trick. 🤷‍♀️

**Pull request closed without merging**